common/protob/messages-bitcoin.proto

```diff
@@ -110,6 +110,7 @@ message GetAddress {
     optional MultisigRedeemScriptType multisig = 4;                     // filled if we are showing a multisig address
     optional InputScriptType script_type = 5 [default=SPENDADDRESS];    // used to distinguish between various address formats (non-segwit, segwit, etc.)
     optional bool ignore_xpub_magic = 6;                                // ignore SLIP-0132 XPUB magic, use xpub/tpub prefix for all account types
+    optional bool chunkify = 7 [default=true];                          // display the address in chunks of 4 characters
```

Comment:
We don't want this to default to true on the protobuf level, do we? 

Or in another word, we would use `default=true` only if the old behaviour was chunkifying by default and we are introducing an option to turn it off. But it's the other way around - the old behaviour did not chunkify be default.

```suggestion
    optional bool chunkify = 7;                                         // display the address in chunks of 4 characters
```

OTOH it's fine to default this value on higher levels - for example trezorlib cli.

Reply:
Right, in the end, I will probably default it to `false` here, but for the development process, it is nice to have `true` here so I do not have to change tests and see the chunkifying on all the possible places.

Reply:
Now I do not have any default value on the protobuf side